proposals/lip-0050.md

```diff
@@ -225,10 +302,32 @@ reclaimParamsSchema = {
 ```python
 def verify(trs: Transaction) -> None:
     legacyAddress = getLegacyAddress(trs.senderPublicKey)
-    if legacyAccounts(legacyAddress) does not exists :
+    if legacyAccounts(legacyAddress) is empty:
+        address = SHA-256(trs.senderPublicKey)[:NUM_BYTES_ADDRESS]
+        emitPersistentEvent(
+            module = MODULE_NAME_LEGACY,
+            name = EVENT_NAME_RECLAIM_FAILED,
+            data = {
+                "legacyAddress": legacyAddress,
+                "address": address,
+                "reason": RECLAIM_FAILED_NO_LEGACY_ACCOUNT
+            },
+            topics = [legacyAddress, address]
+        )
```

Comment:
I don't think we need emit events during verification(s). If verification fails, the command never goes in the block chain.

Reply:
Indeed, removed events from verification.